engine/settings/base.py

```diff
@@ -480,6 +480,10 @@ class BrokerTypes:
         "task": "apps.mobile_app.tasks.conditionally_send_going_oncall_push_notifications_for_all_schedules",
         "schedule": 10 * 60,
     },
+    "notify_shift_swap_requests": {
+        "task": "apps.mobile_app.tasks.notify_shift_swap_requests",
```

Comment:
this also needs to be added to [`CELERY_TASK_ROUTES`](https://github.com/grafana/oncall/blob/dev/engine/settings/prod_without_db.py#L47) (and also in `oncall-private`), right?

Reply:
added in https://github.com/grafana/oncall/pull/2717/commits/747f3d7a83464440930ffd5850aba97ebc105134